lunatik.h

```diff
@@ -322,6 +322,12 @@ do {									\
 	lua_pop(L, 1);							\
 } while (0)
 
+static inline unsigned int lunatik_checkuint(lua_State *L, int idx)
+{
+	lua_Integer __val = luaL_checkinteger(L, idx);
```

Comment:
The overall patch is looking good to me.. I would just remove the `__` prefix unless we have a good reason for adding it